enterprise/app/usage/usage.tsx

```diff
@@ -93,3 +95,7 @@ export default class UsageComponent extends React.Component<UsageProps, State> {
     );
   }
 }
+
+function formatMinutes(usec: Number): string {
```

Comment:
I think we charge by the second (but you can check in with @gtli7) - so it might be worth showing that

Reply:
Confirmed with George that we charge by minutes